inc/Engine/Common/PerformanceHints/Admin/Controller.php

```diff
@@ -126,7 +126,7 @@ public function truncate_from_admin( $clean ) {
 			'status'  => 'success',
 			'message' => sprintf(
 				// translators: %1$s = plugin name.
-				__( '%1$s: Critical images cleared!', 'rocket' ),
+				__( '%1$s: WP Rocket: Critical images and Lazy Render data was cleared!', 'rocket' ),
```

Comment:
We are already passing the formatted string in the first parameter in `sprintf`
```suggestion
				__( '%1$s: Critical images and Lazy Render data was cleared!', 'rocket' ),
```